### DISCUSSION TITLE
does this webui works on amd gpu?

### DISCUSSION THREAD

**Discussion by doctord98** [oobabooga/text-generation-webui#993] [General]
I am on windows with amd gpu 6600xt does this works on it, as I am not able to make it work, so I guess it only works on nvidia, what about linux, do amd gpus work with this in linux environment? please answer if you know something about this project and amd gpu support?

**Comment by BarfingLemurs** [comment]
its possible, but you will likely need linux. https://rentry.org/eq3hg
It's very tough for now and you need some computer skills.
I want to see if anyone has done it with WSL or not.

**Reply by doctord98** [reply]
Yes tried on wsl, I think my GPU does not support rocm

**Comment by Spencer-Dawson** [comment]
There are multiple people using it with AMD GPUs in Ubuntu. I even wrote an installer for that(well two actually) and the README mentions some AMD install instructions. However AMD GPU support relies on AMD ROCm which does not support Windows so if that is what you are looking for you are probably out of luck.

**Reply by doctord98** [reply]
My GPU does not support rocm, I think or maybe some other problem, I am on wsl tho, so yeah, looks like I am out of luck,

**Reply by BarfingLemurs** [reply]
https://github.com/oobabooga/text-generation-webui/issues/964#issuecomment-1509968610
